lib/MTT/Test/Build/Trivial.pm

```diff
@@ -56,6 +58,8 @@ sub Build {
                        "trivial_tests_cflags");
     my $fflags = Value($ini, $config->{full_section_name}, 
                        "trivial_tests_fflags");
+    my $f08flags = Value($ini, $config->{full_section_name}, 
+                       "trivial_tests_f08flags");
```

Comment:
If we do this, we should probably do it properly: add mpifh_flags, usempi_flags, and usempif08_flags.

We can probably leave fflags -- those can be flags that apply to all fortran programs, but then you can use the individual flags vars to specify flags for specific bindings.